docs/virtual_machines/istio_service_mesh.md

```diff
@@ -0,0 +1,92 @@
+# Istio service mesh
+
+Service mesh allows to monitor, visualize and control traffic between microservices.
+Kubevirt supports running VMs as a part of Istio service mesh.
+
+## Limitations
+
+- Istio service mesh is only supported with a pod network masquerade binding.
+
+- Istio uses a [list of ports](https://istio.io/latest/docs/ops/deployment/requirements/#ports-used-by-istio) for it's own purposes, these ports must not
+  be explicitly specified in a VMI interface.
+
+## Prerequisites
+
+This guide assumes that Istio is already deployed and uses Istio CNI Plugin. See [Istio documentation](https://istio.io/latest/docs/) for more information.
+
+## Create a VirtualMachineInstance with enabled Istio proxy injecton
+
+If your cluster uses Multus to manage CNI, the following NetworkAttachmentDefinition is required in the application namespace:
```

Comment:
Is not this a part of the prerequisite? If it is, maybe it should be a part of troubleshooting if you consider this a common mistake? I won't fight for this

Reply:
Moved this to Prerequisites section and also added a case to the Troubleshooting section describing what happens when the istio-cni NAD is missing